content/courses/program-optimization/program-configuration.md

```diff
@@ -422,25 +419,25 @@ We'll quickly learn while testing our program that it could benefit from the
 flexibility provided by an admin-controlled configuration account and some
 feature flags.
 
-### 1. Starter
+#### 1. Starter
 
-Download the starter code from
-the [`starter` branch of this repository](https://github.com/solana-developers/admin-instructions/tree/starter).
-The code contains a program with a single instruction handler and a single test
-in the `tests` directory.
+Download the starter code from the `starter` branch
+of [this repository](https://github.com/Unboxed-Software/solana-admin-instructions/tree/starter).
```

Comment:
Why did you change this? Per the Superteam Earn instructions were are retiring the unboxed repos. That repo has moved, see the README on https://github.com/Unboxed-Software/solana-admin-instructions/

Reply:
Actually ignore that it was for the wrong lesson.